go.mod

```diff
@@ -446,9 +446,12 @@ replace (
 	github.com/ethereum-optimism/optimism => github.com/babylonlabs-io/optimism v1.8.1-0.20240901154247-a38e38a679ab
 	github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101408.0-rc.4.0.20240827042333-110c433a2469
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	// fix for `make build` error: invalid CFI advance_loc expression
+	github.com/supranational/blst => github.com/supranational/blst v0.3.13
 	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
-
 	// avoid v1.66 that has a breaking change for protobuf. That change breaks the relayer.
 	// https://github.com/grpc/grpc-go/issues/7569
 	google.golang.org/grpc => google.golang.org/grpc v1.65.0
+	// fix for `go mod tidy` error: unknown revision v1.8.6
+	nhooyr.io/websocket => github.com/coder/websocket v1.8.7
```

Comment:
just curious, why previous CI run never detect these?

Reply:
yes, i tested on a new server with ubuntu 22.04, and it works fine.

for `go mod tidy`, i saw it automatically selects the appropriate version on this new server
```
go: downloading nhooyr.io/websocket v1.8.17
```
for `make build`, the error only occurs on x86 macOS using LLVM 17

so i will revert these changes.